package.json

```diff
@@ -22,7 +22,6 @@
     "google-auth-library": "^0.10.0",
     "googleapis": "^16.0.0",
     "lighthouse": "1.6.0",
-    "median": "^0.0.2",
```

Comment:
Just committed a my proposal for the code change, when you have a minute let me know if you think it's ok @paulirish